ompi/mca/osc/ucx/osc_ucx_comm.c

```diff
@@ -350,23 +350,27 @@ static inline int get_dynamic_win_info(uint64_t remote_addr, ompi_osc_ucx_module
     }
 
     memcpy(&win_count, temp_buf, sizeof(uint64_t));
-    assert(win_count > 0 && win_count <= OMPI_OSC_UCX_ATTACH_MAX);
+    if (win_count > 0 && win_count <= OMPI_OSC_UCX_ATTACH_MAX) {
```

Comment:
I think this should be:

```suggestion
    if (win_count < 0 || win_count > OMPI_OSC_UCX_ATTACH_MAX) {
```

Reply:
oh jeez, of course

Reply:
actually not quite, right.
Should be: `if (win_count <= 0`
Fixed in latest force push - I think we should be good now